### PR TITLE
Show P chain balances correctly

### PR DESCRIPTION
**Pull request by Ayagoumi**
Changing Balance Breakdown to also support the new values Deposited (P), Bonded (P), and Bonded & Deposited (P) based on the selected network

### CODE REVIEW

File: src/components/wallet/TopCards/BalanceCard/BalanceCard.vue [hunk 1]
```diff
@@ -121,28 +117,48 @@ import UtxosBreakdownModal from '@/components/modals/UtxosBreakdown/UtxosBreakdo
...
         utxos_modal: UtxosBreakdownModal
     }
 
+    mounted() {
+        if (this.depositAndBound) this.$store.dispatch('Assets/getPChainBalances')
+    }
+
+    @Watch('$store.state.Network.selectedNetwork.networkId')
+    SupportdepositAndBound(): void {
+        console.log(
```
Comment (Chriserus) [inline src/components/wallet/TopCards/BalanceCard/BalanceCard.vue]: Remove console logs please

File: src/components/wallet/TopCards/BalanceCard/BalanceCard.vue [hunk 2]
```diff
@@ -227,26 +243,39 @@ export default class BalanceCard extends Vue {
         return ''
     }
 
+    // Locked balance is the sum of locked CAM tokens P chain (bonded + deposited + bondedAndDeposited)
     // Locked balance is the sum of locked AVAX tokens on X and P chain
```
Comment (Chriserus) [inline src/components/wallet/TopCards/BalanceCard/BalanceCard.vue:241]: Remove old description

**Review by Chriserus — APPROVED** [review]
LGTM

File: src/components/wallet/TopCards/BalanceCard/BalanceCard.vue (hunk 2 above)
Comment (Ayagoumi) [reply src/components/wallet/TopCards/BalanceCard/BalanceCard.vue:241]: both descriptions are valid, we still support the avalanche way, and also our method of calculating the total balance

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**